### ISSUE TITLE
Full Accessibility & 508 Office Audit [Public Websites - Campaign Landing Page]

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#20436]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(VSP understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [x] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1) in Slack; tag @Trevor Pierce
- [ ] VSP: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] VSP: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)

## Artifacts
- Access information for the tool or feature in staging. [CLP Preview](http://preview-prod.vfs.va.gov/preview?nodeId=15068)
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
- List of known issues:  @joshkimux did an initial review for our design as we wanted to validate it was still ally compliant that ticket is here #18756.  **_Josh identified the need for video transcripts and have that ticket for review and work in future sprints post MVP launch._** 

## Additional information
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by brianalloyd** [comment]
Placing in backlog, @1Copenut `NO ACTION` needed at this time until we've cleared VSP Staging review scheduled for March 4, 2021.

**Comment by laflannery** [comment]
No issues attached to this, closing
